### ISSUE TITLE
Logo should not be an H1

### ISSUE THREAD

**Issue by michellekondou** [alistapart/AListApart#326]
@awkawk @aparkening Following a discussion we had with @murtaugh concerning nested headed order, Tim expressed the concern that maybe the logo should not be an H1. I did a bit of research and I think it's a good idea. On the homepage (where there is no clear top-level heading) we could mark it up as an H1, elsewhere just a paragraph. The issue is referenced extensively in the Financial Times repo https://github.com/Financial-Times/n-header-footer/issues/39 with many useful examples of other sites using this approach. What does everyone think?

**Comment by aparkening** [comment]
@michellekondou: +1, I think taking the logo out of H1 on sub pages is great.

**Comment by awkawk** [comment]
Is there an H1 on the sub pages then? Whatever structure people think makes sense if fine by me.

**Comment by michellekondou** [comment]
Yes, the H1 on each subpage will be relative to the content of that page. For example, in the article template it will be the article heading. In the about section it will be "About" and so on.
